### PR TITLE
Fixing embargo visibility issues

### PR DESCRIPTION
**Pull request by rawOrlando**
Fixes bug [TDLR-2423](https://tuftswork.atlassian.net/browse/TDLR-2423)

Fixes issues where objects and associated files ingested didn't change visibility public after embargo expired.

### CODE REVIEW

File: app/lib/tufts/import_record.rb [hunk 1]
```diff
@@ -110,12 +110,28 @@ def title
 
     def visibility_during_embargo
       return nil if metadata.nil?
-      metadata.xpath('./tufts:visibility_during_embargo', mapping.namespaces).children.map(&:content).first
+      given_value = metadata.xpath('./tufts:visibility_during_embargo', mapping.namespaces).children.map(&:content).first
```
Comment (mkorcy) [inline app/lib/tufts/import_record.rb:113]: why is this called `given_value` and not `visibility_during_embargo`?

File: app/lib/tufts/import_record.rb [hunk 2]
```diff
@@ -110,12 +110,28 @@ def title
...
+
+      # Default to private
+      Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
     end
 
     def visibility_after_embargo
       return nil if metadata.nil?
-      metadata.xpath('./tufts:visibility_after_embargo', mapping.namespaces).children.map(&:content).first
+
+      given_value = metadata.xpath('./tufts:visibility_after_embargo', mapping.namespaces).children.map(&:content).first
```
Comment (mkorcy) [inline app/lib/tufts/import_record.rb:125]: why is this called `given_value` and not `visibility_after_embargo`?

File: spec/factories/pdf.rb [hunk 3]
```diff
@@ -37,4 +37,28 @@
...
+    rights_statement { ['http://bostonhistory.org/photorequest.html'] }
+    date_created { ['1999'] }
+    publisher { [FFaker::Company.name, FFaker::Company.name] }
+    description { [FFaker::HipsterIpsum.paragraph] }
+    abstract { [FFaker::HipsterIpsum.paragraph] }
+    creator { [FFaker::Name.name, FFaker::Name.name] }
+    corporate_name { [FFaker::Company.name] }
+    contributor { [FFaker::Name.name] }
+    after(:build) { |work, evaluator| work.apply_embargo(evaluator.embargo_date, evaluator.current_state, evaluator.future_state) }
+    after(:create) { |work, evaluator| 2.times { work.ordered_members << FactoryBot.create(:file_set, user: evaluator.user) } }
```
Comment (mkorcy) [inline spec/factories/pdf.rb:62]: why 2 times?

**Review by mkorcy — COMMENTED** [review]
i just had some questions as I read through it.

File: app/lib/tufts/import_record.rb (hunk 2 above)
Comment (rawOrlando) [reply app/lib/tufts/import_record.rb:125]: I struggled to come up with a variable name here that's, better, will change.

File: app/lib/tufts/import_record.rb (hunk 1 above)
Comment (rawOrlando) [reply app/lib/tufts/import_record.rb:113]: I struggled to come up with a variable name here that's, better, will change.

File: spec/factories/pdf.rb (hunk 3 above)
Comment (rawOrlando) [reply spec/factories/pdf.rb:62]: You help out here and wrote this code. I general I think its a good idea to have two file sets associated so I left it be.

**Comment by rawOrlando** [comment]
@mkorcy I have updated the variable names, what do you want to do about the factory?

**Review by mkorcy — APPROVED** [review]
Looks good, and slightly increases the test coverage, thanks.

**Pull request merged**